src/hooks/useBorrowRepayModal/Modal/Repay/index.tsx

```diff
@@ -207,24 +207,27 @@ export const RepayForm: React.FC<RepayFormProps> = ({
 };
 
 export interface RepayProps {
-  asset: Asset;
+  token: Token;
+  vToken: Token;
   includeXvs: boolean;
   onClose: () => void;
 }
 
-const Repay: React.FC<RepayProps> = ({ asset, onClose, includeXvs }) => {
+const Repay: React.FC<RepayProps> = ({ token, vToken, onClose, includeXvs }) => {
```

Comment:
nit: seems like we could use just the token asset and get the related vToken from the asset object but I didn't dig deep into that idea

Reply:
I want to keep this as modular as possible, and asset has a lot of fields that aren't all relevant to this component, so we're better off passing token and vToken and letting the component fetch the data it needs

Reply:
I mistyped and meant to just write token. I was suggesting removing a prop rather than adding one

Reply:
but yeah from your previous comment it sounds like asset would work here too in place of both